management/azure_mgmt_compute/lib/2016-03-30/generated/azure_mgmt_compute.rb

```diff
@@ -16,133 +16,131 @@
 require 'faraday-cookie_jar'
 require 'concurrent'
 require 'ms_rest'
-require 'generated/azure_mgmt_compute/module_definition'
+require '2016-03-30/generated/azure_mgmt_compute/module_definition'
 require 'ms_rest_azure'
 
-module Azure::ARM::Compute
-  module Api_2016_03_30
-  autoload :AvailabilitySets,                                   'generated/azure_mgmt_compute/availability_sets.rb'
-  autoload :VirtualMachineExtensionImages,                      'generated/azure_mgmt_compute/virtual_machine_extension_images.rb'
-  autoload :VirtualMachineExtensions,                           'generated/azure_mgmt_compute/virtual_machine_extensions.rb'
-  autoload :VirtualMachineImages,                               'generated/azure_mgmt_compute/virtual_machine_images.rb'
-  autoload :UsageOperations,                                    'generated/azure_mgmt_compute/usage_operations.rb'
-  autoload :VirtualMachineSizes,                                'generated/azure_mgmt_compute/virtual_machine_sizes.rb'
-  autoload :VirtualMachines,                                    'generated/azure_mgmt_compute/virtual_machines.rb'
-  autoload :VirtualMachineScaleSets,                            'generated/azure_mgmt_compute/virtual_machine_scale_sets.rb'
-  autoload :VirtualMachineScaleSetVMs,                          'generated/azure_mgmt_compute/virtual_machine_scale_set_vms.rb'
-  autoload :ContainerServices,                                  'generated/azure_mgmt_compute/container_services.rb'
-  autoload :ComputeManagementClient,                            'generated/azure_mgmt_compute/compute_management_client.rb'
+module Azure::ARM::Compute::Api_2017_01_31
```

Comment:
This seems odd. Namespace supposed to be `Azure::ARM::Compute::Api_2016_03_30`. Isn't it?

Reply:
Good catch, this is because I messed up here; https://github.com/Azure/azure-sdk-for-ruby/pull/925/files#diff-06b2d3b23dce96e1619d2b53d6c947ecR74 
this stuff comes from configuration, so if we make a mistake there, we'll see it here. We'll have to review the metadata file (manual or autogenerated)